.github/workflows/ci.yml

```diff
@@ -93,7 +93,7 @@ jobs:
           target
         key: ${{ runner.os }}-${{ runner.arch }}
     - name: Install Rust
-      uses: dtolnay/rust-toolchain@nightly
+      uses: dtolnay/rust-toolchain@stable
```

Comment:
It looks like this is changing the overall build to be stable, not the examples. I'm not sure if it makes sense to switch the whole project over to stable at this time. If the goal is to build only some examples with the stable toolchain, maybe it makes sense to make a copy of the examples section that runs only the ones that work on stable?

Reply:
First, I don't think it's necessary to keep using nightly. It's good to encourage users to use stable.
Second, the CI job didn't work. I will make them all work with the stable version.

Reply:
If you do end up changing each of the examples, it might be helpful to split each one into a separate PR. Easier to review and merge that way.